harvdev_utils/psycopg_functions/fb_feature_classes.py

```diff
@@ -587,22 +596,22 @@ def get_agr_floc(self):
             # Assign AGR type only if insertion position is precise/unambiguous.
             if self.agr_stop - self.agr_start == 1:
                 self.agr_type = 'SO:0000667'
-        elif self.feature_type == 'insertion_site':
-            # For rare non-CRIMIC "insertion_site" features, floc interpretation is unclear/variable - ignore.
-            # But for CRIMIC insertions, it does represent the insertion/delin.
-            # Assumption here is that insertion is before reported onbase fmax. Need to clarify.
-            if self.name.startswith('TI{CRIMIC'):
-                # Simple insertion.
-                if self.fmax - self.fmin == 1:
-                    self.agr_type = 'SO:0000667'
-                    self.agr_start = self.fmin
-                    self.agr_stop = self.fmax
-                # A delin.
-                # Odd thing is that 
-                elif self.fmax - self.fmin > 1:
-                    self.agr_type = 'SO:1000032'
-                    self.agr_start = self.fmin + 1
-                    self.agr_stop = self.fmax - 1
+        # Need to reconsider portion below and split out this function into smaller less complex ones.
```

Comment:
Not sure why this bit is commented out. Says it needs to be split out into smaller bits, but do not see this being done? If it is now somewhere else then this needs to be deleted and not commented out. If not done yet then has this been missed?

Reply:
This bit of code was started, but not completed, and is not currently in use. Commented this out to pass flake8, but left the code in there as a reminder for when I can get back to it. I can delete it if that makes sense.

Reply:
Was not sure if it was something you had just forgotten or not. As this is obviously okay, happy for you to resolve it.